docs/DeveloperGuide.adoc

```diff
@@ -131,6 +150,20 @@ As a more OOP model, we can store a `Tag` list in `Address Book`, which `Person`
  +
 image:BetterModelClassDiagram.png[]
 
+[[Design-Game]]
+=== Game component
+
+
+.Structure of the Game Component
+image::GameClassDiagram.png[]
```

Comment:
A method should not be indicated in the association label
<img width="570" alt="Screenshot 2019-10-24 at 1 50 57 PM" src="https://user-images.githubusercontent.com/27060502/67457133-62ba0a00-f665-11e9-83bb-a222a2a9d916.png">